### ISSUE TITLE
False positive for overridden_super_call

### ISSUE THREAD

**Issue by jdmarshall90** [realm/SwiftLint#1360]
```swift
class MyView: UIView {
    override func updateConstraints() {
        defer { super.updateConstraints() }
        
        guard condition == true else { return }

        // add constraints ...
    }
}
```

Or

```swift
class MyVC: UIViewController {
    override func viewDidLoad() {
        defer { super.viewDidLoad() }
    }
}
```
Both trigger `overridden_super_call`.

**Comment by marcelofabri** [comment]
Duplicate of #1301.
